network-config-analyzer/CalicoNetworkPolicy.py

```diff
@@ -79,12 +79,19 @@ def __eq__(self, other):
             self.order == other.order
 
     def __lt__(self, other):  # required so we can evaluate the policies according to their order
+        if not isinstance(other, NetworkPolicy):
+            return False
         if isinstance(other, CalicoNetworkPolicy):
-            if self.order is None:
+            # If not specified "order" defaults to infinity, so policies with unspecified "order" will be applied last.
```

Comment:
See if it's possible to implement this method by just looking at the policy's `get_order()`, and without checking its type

Reply:
right, done.